rca/project_styleguide/templates/patterns/organisms/programme_tabs/experience.html

```diff
@@ -0,0 +1,80 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<section class="section section--start section--end bg bg--dark">
+    <header class="section__header grid">
+        <h2 class="section__heading section__heading--primary heading heading--two">Experience</h2>
+    </header>
+    <div class="section__content">
+        
+        {% if page.experience_introduction or page.experience_cta_link %}
+            <div class="overview grid">
+                {% if page.experience_introduction %}
+                    <div class="overview__item">
+                        <div class="overview__introduction introduction">
+                            {{ page.experience_introduction }}
+                        </div>
+                    </div>
+                {% endif %}
+                
+                {% if page.experience_cta_link %}
+                    <div class="overview__item">
+                        <div class="overview__item-action">
+                            {% for block in page.experience_cta_link %}
+                                {% include_block block %}
+                            {% endfor %}
+                        </div>
+                    </div>
+                {% endif %}
+            </div>
+        {% endif %}
+
+        {% if page.experience_content %}
+            {% for block in page.experience_content %}
+                {% if block.block_type == "social_embeds" %}
+                    <div class="section__row grid">
+                        <div class="results layout__start-one layout__span-two layout__@large-start-two layout__@large-span-three">
+                            {% for embed_block in block.value %}
+                                <div class="results__item layout__span-two layout__@medium-span-one" style="margin-right: 0.5rem;">
+                                    <div class="embed-container">
+                                        {% include_block embed_block %}
+                                    </div>
+                                </div>
+                            {% endfor %}
+                        </div>
+                    </div>
+                {% endif %}
+            {% endfor %}
+            
+            <div class="section__row grid">
+                <div class="results layout__start-one layout__span-two layout__@large-start-two layout__@large-span-three">
+                    {% for block in page.experience_content %}
```

Comment:
This will force all story content blocks to the bottom. We can have any combination so it could be 

1. Embeds
2. Stories
3. Embeds

or 

1. Stories
2. Stories
3. Embeds

etc.

Reply:
fixed in https://github.com/torchbox/rca-wagtail-2019/pull/1094/commits/5d3c097d023c4b3c9903b996af6de707b560c15e